### ISSUE TITLE
As Jess, I need to configure which Task screen a post-onboarding question show up on

### ISSUE THREAD

**Issue by aloverso** [newjersey/business.nj.gov#345]
...so that I can make it more accurate

**Comment by aloverso** [comment]
@Jesslax this is now configurable as an optional property that can be added to any task. I've added it now to [building-permit](https://github.com/newjersey/business.nj.gov/blob/main/web/roadmaps/tasks/building-permit.md)

You can see that there is a line: `postOnboardingQuestion: "construction-renovation"`

You can add this to any task, and the value references a filename in the [post-onboarding content folder](https://github.com/newjersey/business.nj.gov/tree/main/web/display-content/post-onboarding)

If we want to add more post-onboarding questions, that would still have to be a new code-change, but you can modify this one and move it between tasks as much as necessary

**Comment by skeeJay** [comment]
@Jesslax QA to you, on both this and #113, which was blocked by this. cc @kmelefsky

**Comment by Jesslax** [comment]
@aloverso Is it possible to put the question wherever I want within the task? For example, right now when I look at building-permit, it asks the question at the bottom of the task screen, but I'd prefer if the question and related radio buttons replaced the following text, "If you are planning to renovate your site, this task may apply to you. Consider completing the following activities before starting any renovation or construction:"

**Comment by aloverso** [comment]
@Jesslax mind if we open a new story for that? it's definitely doable though

**Comment by Jesslax** [comment]
@skeeJay @aloverso I can make a new one, and we can close this tkt, but I wouldn't take down #113 until the post onboarding questions are in a location that is useful for Poppy.

**Comment by aloverso** [comment]
Created #371
